client/app/templates/components/projects-map-data.hbs

```diff
@@ -11,11 +11,6 @@
   {{map.on 'click' (action 'handleMapClick')}}
   {{map.on 'mousemove' (action 'handleMouseMove')}}
 
-  {{#if highlightedFeature}}
-    <div class='map-tooltip' style="top:{{tooltipPoint.y}}px;left:{{tooltipPoint.x}}px;">
-      {{highlightedFeature.properties.dcp_projectname}} (<DateDisplay @date={{highlightedFeature.properties.dcpLastmilestonedate}} @outputFormat="MMMM, YYYY" />)
```

Comment:
It's a little disturbing that 2 columns were dropped from the dataset and not communicated to the OSE (or w/e the team was called when this happened) so we could at least update the codebase.

Reply:
agreed – these bugs will def keep cropping up unless we find a better way to communicate data changes (other than not at all)